### PR TITLE
Lofreq 2.1.4

### PR DESCRIPTION
**Pull request by JohnGreeley**
Unblacklisting lofreq and updating to 2.1.4
----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/lofreq/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu -o pipefail
+
+./configure --with-htslib=${PREFIX}/lib --prefix=${PREFIX}
```
Comment (jmarshall) [inline recipes/lofreq/build.sh]: That's not doing anything for you; it should be just `--with-htslib=${PREFIX}`. But I would suggest following the samtools and bcftools recipes and using `--with-htslib=system`.

Comment (JohnGreeley) [reply recipes/lofreq/build.sh]: Thank you, I'm still learning the best practice with these libraries, and I struggle with the myriad settings for compiling complex C/C++ packages like these.

Comment (jmarshall) [reply recipes/lofreq/build.sh]: Sorry, I should have been more explicit: keep the `--prefix` too.

```
./configure --with-htslib=system --prefix=$PREFIX
```

File: recipes/lofreq/meta.yaml [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-  number: 1
-  script: "mkdir -p $PREFIX/bin; chmod a+x bin/*; cp bin/* $PREFIX/bin"
-  skip: False
+  number: 0
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
     - python
+    - htslib
```
Comment (dpryan79) [inline recipes/lofreq/meta.yaml]: Is htslib being statically compiled in? If not it will be needed during runtime.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/lofreq/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/lofreq/meta.yaml]: Ah, apparently we changed the htslib recipe to auto export itself, that's nice.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

Thanks @JohnGreeley

**Pull request merged**